onelogin/get.go

```diff
@@ -216,3 +199,52 @@ func Get(app, provider string) (*awsprovider.Credentials, error) {
 
 	return &creds, nil
 }
+
+// getDevice returns the MFA device used by the user. If there is
+// more than one available the user is prompted which one should
+// be used.
+func getDevice(devices []Device) (deviceID, deviceType string, err error) {
+	if len(devices) == 0 {
+		// this should never happen
+		err = errors.New("No MFA device returned by Onelogin")
+		return
+	}
+	if len(devices) == 1 {
+		deviceID = fmt.Sprintf("%v", devices[0].DeviceId)
+		deviceType = devices[0].DeviceType
+		return
+	}
+
+	var selection int
+	for {
+		for i, d := range devices {
+			fmt.Printf("%d. %d - %s\n", i+1, d.DeviceId, d.DeviceType)
+		}
+
+		fmt.Printf("Please choose an MFA device to authenticate with (1-%d): ", len(devices))
+		var input string
+		_, err := fmt.Scanln(&input)
+		if err != nil {
+			fmt.Printf("Error reading input: %v\n", err)
+			continue
+		}
+
+		// Verify we got an integer.
+		selection, err = strconv.Atoi(input)
+		if err != nil {
+			fmt.Printf("Invalid input '%s'\n", input)
+			continue
+		}
+
+		// Verify selection is within range.
+		if selection < 1 || selection > len(devices) {
+			fmt.Printf("Invalid MFA device selected\n")
```

Comment:
Why not be consistent with the [role selection loop](https://github.com/allcloud-io/clisso/blob/0a2f6cf4becb4d1c3b21bf889ddf0729d8532944/saml/saml.go#L124)?

```go
fmt.Printf("Invalid value %d. Valid values: 1-%d\n", selection, len(devices))
```

I'm not sure this phrasing is ideal, but if we change one we should change the other, too. No point in having two different phrasings in my opinion.